index.js

```diff
@@ -40,6 +40,7 @@ export default class IDBStorage {
                 this.opening = null
                 this.db = db
                 this.db.onversionchange = () => this.close()
+                this.db.onclose = () => this.db = null
```

Comment:
This seems to be the single change you did. I wonder why the built files have all the other changes in the quotes. Just curious, not a big deal.

Reply:
Why don't we call `this.close()` here?

Reply:
I'm not on top of how this library works, but just basing my analysis on the names, I assume that "onclose" is called when the connection was closed. Not sure if it makes sense to call `this.close()` when that happens.

Reply:
> I assume that "onclose" is called when the connection was closed.

The `close` event is only triggered when the database is "unexpectedly closed":
_"The close event is fired on IDBDatabase when the database connection is unexpectedly closed. This could happen, for example, if the underlying storage is removed or if the user clears the database in the browser's history preferences."_[*](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase/close_event)

> Why don't we call this.close() here?

That's prettier. Updated. 👍

> I wonder why the built files have all the other changes in the quotes.

Also didn't understand why. Nothing in my editor config appears to be responsible for this. Assumed this was babel's doing. 🤷‍♂️